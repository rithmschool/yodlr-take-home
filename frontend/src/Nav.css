  .Nav a {
    text-decoration: none;
    color: black;
    margin: auto 1rem;
}

.Nav a.active{
    font-weight: bold;
}