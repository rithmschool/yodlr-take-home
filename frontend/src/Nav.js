import React from 'react';
import { NavLink } from 'react-router-dom';
import './Nav.css';


const Nav = () => {

  return (
    <nav className="Nav navbar navbar-light bg-white">
      <span className="navbar-brand">Yodlr</span>
      <span>
        <NavLink exact to="/">Home</NavLink>
        <NavLink exact to="/signup">Sign Up</NavLink>
        <NavLink exact to="/admin">Admin</NavLink>
      </span>
    </nav>
  )
}


export default Nav;