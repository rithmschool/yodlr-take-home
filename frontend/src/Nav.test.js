import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Nav from './Nav';


// smoke test
it("renders without crashing", function() {
  render(<MemoryRouter><Nav /></MemoryRouter>);
});

// snapshot test
it("matches snapshot", function() {
  const { asFragment } = render(<MemoryRouter><Nav /></MemoryRouter>);
  expect(asFragment()).toMatchSnapshot();
});
