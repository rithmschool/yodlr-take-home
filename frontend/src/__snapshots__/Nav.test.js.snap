// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="Nav navbar navbar-light bg-white"
  >
    <span
      class="navbar-brand"
    >
      Yodlr
    </span>
    <span>
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        Home
      </a>
      <a
        href="/signup"
      >
        Sign Up
      </a>
      <a
        href="/admin"
      >
        Admin
      </a>
    </span>
  </nav>
</DocumentFragment>
`;
